<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldWindow</class>
 <widget class="QMainWindow" name="WorldWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="FileNameLabel">
      <property name="text">
       <string>InputFile</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="FileNameEdit"/>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="ExitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="AnalyzeButton">
      <property name="text">
       <string>Analyze</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="SearchButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="1" column="0">
        <widget class="QRadioButton" name="TypeFFT">
         <property name="text">
          <string>FFT</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="TypeStar">
         <property name="text">
          <string>STAR</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Spectrum Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>F0 View</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="DioEnabled">
         <property name="text">
          <string>Show F0 by DIO</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="TypeLPC">
         <property name="text">
          <string>LPC</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="DioAllEnabled">
         <property name="text">
          <string>Show All results of DIO</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QScrollArea" name="ViewerArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="SpecgramView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>395</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AnalyzeButton</sender>
   <signal>clicked()</signal>
   <receiver>WorldWindow</receiver>
   <slot>beginAnalyze()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchButton</sender>
   <signal>clicked()</signal>
   <receiver>WorldWindow</receiver>
   <slot>searchDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TypeFFT</sender>
   <signal>toggled(bool)</signal>
   <receiver>WorldWindow</receiver>
   <slot>changeSpectrumType(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TypeStar</sender>
   <signal>toggled(bool)</signal>
   <receiver>WorldWindow</receiver>
   <slot>changeSpectrumType(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TypeLPC</sender>
   <signal>clicked(bool)</signal>
   <receiver>WorldWindow</receiver>
   <slot>changeSpectrumType(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>WorldWindow</receiver>
   <slot>playSound()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>beginAnalyze()</slot>
  <slot>searchDirectory()</slot>
  <slot>changeSpectrumType(bool)</slot>
  <slot>playSound()</slot>
 </slots>
</ui>
